pkg/server/service/loadbalancer/p2c/p2c_test.go

```diff
@@ -0,0 +1,384 @@
+package p2c
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/traefik/traefik/v3/pkg/config/dynamic"
+	"github.com/traefik/traefik/v3/pkg/server/service/loadbalancer"
+)
+
+func TestP2C(t *testing.T) {
+	testCases := []struct {
+		name            string
+		handlers        []*namedHandler
+		rand            *mockRand
+		expectedHandler string
+	}{
+		{
+			name:            "oneHealthyHandler",
```

Comment:
For the sake of consistency could you please use a `desc` property to describe the test case as usual?